hadoop-ozone/s3gateway/src/test/java/org/apache/hadoop/ozone/s3/endpoint/TestBucketHead.java

```diff
@@ -66,8 +65,8 @@ public void testHeadFail() throws Exception {
     try {
```

Comment:
nit: please change this to `assertThrow()`.

Reply:
Done, thanks.